[BE] minor improvement to dist quantization

wanchaol opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #66649
* #66581

some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @SciPioneer @H-Huang

Commit:
[BE] minor improvement to dist quantization

some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

[ghstack-poisoned]

Commit:
Update on "[BE] minor improvement to dist quantization"


some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc pietern mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse @SciPioneer @H-Huang

[ghstack-poisoned]

cbalioglu reviewed (approved):
One minor comment; otherwise, looks good.

Commit:
Update on "[BE] minor improvement to dist quantization"


some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc pietern mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse @SciPioneer @H-Huang

[ghstack-poisoned]

Commit:
Update on "[BE] minor improvement to dist quantization"


some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc pietern mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse @SciPioneer @H-Huang

[ghstack-poisoned]

Commit:
Update on "[BE] minor improvement to dist quantization"


some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc pietern mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse @SciPioneer @H-Huang

[ghstack-poisoned]

Commit:
Update on "[BE] minor improvement to dist quantization"


some minor changes to dist quantization, mainly change the namespace and add some notes for future code dedup

Differential Revision: [D31663043](https://our.internmc.facebook.com/intern/diff/D31663043/)

cc pietern mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse @SciPioneer @H-Huang

[ghstack-poisoned]
